form.html

```diff
@@ -21,8 +21,9 @@ <h2 class="company">Coders Building Communities</h2><span><img class="logo" src=
             <h2 class="test-title hidden">No Test Selected</h2>
 
             <div id="slide-overview" class="questionnaire-slide slide-overview">
-                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Pariatur, maxime quis sunt et  <br>cumque nostrum tempore amet commodi
-                    possimus mollitia sit <br> minima impedit molestiae tempora tenetur fuga. Unde, quas numquam!</p>
+                <p>We are a group of programers and web developers looking to
+                    help our communities while building a better web one site at a time.
+                     <br> minima impedit molestiae tempora tenetur fuga. Unde, quas numquam!</p>

```

Comment:
Instead of using a \<br\> in a \<p\> element, start a new \<p\>. It makes it easier for screen readers.

Reply:
The <br> was there already I left it in the event someone else had something planned for that area :)

Reply:
GitHub treats HTML tags as *HTML tags* so you can make fancy comments. To escape the formatting, add a backslash \\ to the angle bracket.

\\<p\\> becomes \<p\>